src/UI/Component/ViewControl/Pagination.php

```diff
@@ -121,4 +121,9 @@ public function getDropdownLabel(): string;
 	 * will be translated, a custom label will not.
 	 */
 	public function getDefaultDropdownLabel(): string;
+
+	/**
+	 * Calculate the total number of pages.
```

Comment:
+1 for moving this into the interface;
however, the doc was wrong already and should be updated to something like "Get the current number of entries on this page"